### PR TITLE
Minor changes to Readme 

### PR DESCRIPTION
**Pull request by abhardwaj9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 Sorting filtering and iterating through data using streams
 
-Shows how to do in streams, but also compares it to the pre-streams approach
```
Comment (tp02ga) [inline README.md:5]: Can you update this to say something like:

Shows you how you can use a "functional programming" approach to iterating through Collections with the Java language.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,6 @@
...
\ No newline at end of file
+Shows how to do in streams, but also compares it to the pre-streams approach
+
+Testing for input streams
```
Comment (tp02ga) [inline README.md]: Remove this part

**Review by tp02ga — CHANGES_REQUESTED** [review]
Some changes are requested... See comments

**Review by tp02ga — APPROVED** [review]

**Review by abhardwaj9 — COMMENTED** [review]
Changes made as requested.

**Pull request merged**